### ISSUE TITLE
Make Simple Markdown Editor use monospaced font for "edit mode"

### ISSUE THREAD

**Issue by x80486** [standardnotes/forum#805]
The font used by the `Simple Markdown Editor` when editing is spooky.

I constantly change from that editor and the simple one just to find out that some spacing is messed up due to the font. Is it possible to make it use monospaced when editing, which sort of makes sense across the board?

**Comment by roaet** [comment]
I agree that the markdown editors should respect the Monospace setting unless they are doing internal formatting.

**Comment by tzing** [comment]
would be even better if it could use spellcheck

**Comment by Joss-Steward** [comment]
+1 on this. Markdown editors not using a monospaced font is really weird.  Better yet, maybe the user should be able to choose the font?

**Comment by x80486** [comment]
Friendly semi-annual reminder :rofl: that some users would like to see this implemented :wink:

**Comment by voss** [comment]
Recent and happy convert to Standard Notes, only issue with it so far, is using non monospaced font in the markdown editors. I too support monospace in every markdown editor pane.
